src/chartsFactory.jsx

```diff
@@ -61,7 +61,13 @@ module.exports = function (chartType, Highcharts){
     },
 
     render: function (){
-      let props = this.props;
+      let {
```

Comment:
The only thing I can think of is to store the PropTypes as a separate object so that it can be iterated over to assign properties in props that don't exist in PropTypes to a new object.

```javascript
const componentPropTypes = {
  someProp: PropTypes.string
};

...

render() {
  let propsToPass = {};

  Object.keys(this.props).forEach((key) => {
    if (!componentPropTypes.hasOwnProperty(key)) {
      propsToPass[key] = this.props[key];
    }
  });

  return (
    <div {...propsToPass}/>
  );
}
```

It may be overkill though, if the number of PropTypes is small and unlikely to change frequently.

By the way sorry if that code came through poorly, on my phone right now.